aszczachor/src/components/header/Header.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import { AppBar, Menu, MenuItem, Tab, Tabs, Toolbar } from "@mui/material";
+import FacebookIcon from "@mui/icons-material/Facebook";
+import InstagramIcon from "@mui/icons-material/Instagram";
+import EmailIcon from "@mui/icons-material/Email";
+import { Link } from "react-router-dom";
+
+const pages = [
+  "Editorials",
+  "Videoclip",
+  "Influencers",
+  "Collections",
+  "About",
+];
+
+const Header = () => {
+  const [anchorElement, setAnchorElement] = useState(null);
+  const open = Boolean(anchorElement);
+  const handleClick = event => {
+    setAnchorElement(event.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorElement(null);
+  };
+
+  return (
+    <React.Fragment>
+      <AppBar
+        sx={{
+          background: "#FFFFFF",
+          color: "#000000",
+          position: "sticky",
+        }}
+      >
+        <Toolbar>
+          <Tabs sx={{ marginLeft: "auto", marginRight: "auto" }}>
+            <Tab onClick={handleClick} label="Fashion Styling" />
+            <Tab component={Link} to="/collections" label={pages[3]} />
```

Comment:
Tablica to nie najlepszy pomysł, dodałbym obiekt, w którym trzymamy zarówno path do strony jak i label, coś w stylu:  
```
const routes = {
  collections: {
    path: '/collections',
    label: 'Collections'
  },
 // ... tutaj reszta
}
```